### PR TITLE
Update Readme and the URL for the API documentation

### PR DESCRIPTION
**Pull request by irfanpena**
## Describe Your Changes

- Update Readme to reflect the new updates
- Update the URL for the API doc

## Fixes Issues

- Closes #
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [x] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +101,7 @@ Run the following command to uninstall Cortex globally on your machine:
 
 ```
 # Uninstall globally using NPM
-npm uninstall -g @janhq/cortex
```
Comment (louis-jan) [inline README.md:101]: `npm uninstall -g cortexso`

File: cortex-js/README.md [hunk 2]
```diff
@@ -11,91 +11,39 @@
...
-cortex pull llama3:7b
+Visit [Quickstart](https://cortex.jan.ai/docs/quickstart) to get started.
 
-# Pull a model with the HuggingFace `model_id`
-cortex pull microsoft/Phi-3-mini-4k-instruct-gguf
-```
-3. Load the model:
 ``` bash
-cortex models start llama3:7b
+npm i -g @janhq/cortex
```
Comment (louis-jan) [inline cortex-js/README.md:32]: npm i -g cortexso

File: cortex-js/README.md [hunk 3]
```diff
@@ -141,18 +90,21 @@ The following CLI commands are currently available:
...
 ```
+
 ## Uninstall Cortex
 
 Run the following command to uninstall Cortex globally on your machine:
 
 ```
 # Uninstall globally using NPM
-npm uninstall -g cortexso
+npm uninstall -g @janhq/cortexso
```
Comment (louis-jan) [inline cortex-js/README.md:104]: please update all @janhq/cortex|cortexso to just cortexso

File: cortex-js/README.md [hunk 4]
```diff
@@ -11,91 +11,39 @@
...
-  - 8GB for running up to 3B models.
-  - 16GB for running up to 7B models.
-  - 32GB for running up to 13B models.
-- **VRAM (GPU Mode):**
-
-  - 6GB can load the 3B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-  - 8GB can load the 7B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-  - 12GB can load the 13B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-
-- **Disk**: At least 10GB for app and model download.
```
Comment (louis-jan) [inline cortex-js/README.md:61]: I would love to maintain at least supported OS versions. This is a source of truth, even when the website is down somehow. As a oss user, I don't really like to get redirected to many endpoints.

Comment (irfanpena) [reply cortex-js/README.md:61]: Yeah, sure, I think the prerequisites are important, will put it back in the docs.

**Comment by irfanpena** [comment]
@louis-jan Updated, kindly check again

File: cortex-js/README.md [hunk 5]
```diff
@@ -11,91 +11,49 @@
...
 - **OS**:
   - MacOSX 13.6 or higher.
   - Windows 10 or higher.
-  - Ubuntu 22.04 and later.
-- **RAM (CPU Mode):**
-  - 8GB for running up to 3B models.
-  - 16GB for running up to 7B models.
-  - 32GB for running up to 13B models.
-- **VRAM (GPU Mode):**
+  - Ubuntu 12.04 and later.
```
Comment (louis-jan) [inline cortex-js/README.md:37]: 22.04 and later

Comment (louis-jan) [reply cortex-js/README.md:37]: Required to run Tensorrt-LLM engine (GLIB compatibility)

File: cortex-js/package-lock.json
Comment (louis-jan) [inline cortex-js/package-lock.json:1]: Please discard this file

File: README.md [hunk 6]
```diff
@@ -25,11 +25,21 @@ Cortex currently supports 3 inference engines:
...
+- **OS**:
+  - MacOSX 13.6 or higher.
+  - Windows 10 or higher.
+  - Ubuntu 12.04 and later.
+
+> Visit [Quickstart](https://cortex.jan.ai/docs/quickstart) to get started.
 
 ``` bash
-npm i -g @janhq/cortex
+npm i -g cortexso
```
Comment (louis-jan) [inline README.md:42]: Please help sneak in the following brew steps
https://github.com/janhq/homebrew-cortexso

Comment (louis-jan) [reply README.md:42]: I mean to copy the run steps for brew from that repo README

File: cortex-js/README.md (hunk 5 above)
Comment (irfanpena) [reply cortex-js/README.md]: What do you mean by Required to run Tensorrt-LLM engine (GLIB compatibility)? Is it required for every user to run Tensorrt-LLM engine or only for ubuntu?

Comment (louis-jan) [reply cortex-js/README.md]: I checked with guys, 20.04 for llama.cpp (default), and 22.04 is recommended for every user to run Tensorrt-LLM engine on ubuntu

Comment (louis-jan) [reply cortex-js/README.md]: I mean better leave 22.04 for now (Ubuntu)

Comment (irfanpena) [reply cortex-js/README.md]: I see. Okay, I have updated the README per your feedback. Kindly check it again

File: README.md [hunk 7]
```diff
@@ -98,7 +118,11 @@ Run the following command to uninstall Cortex globally on your machine:
 
 ```
 # Uninstall globally using NPM
-npm uninstall -g @janhq/cortex
+npm uninstall -g cortexso
+
+# Uninstall using Brew
+brew uninstall cortexso
+brew untap janhq/cortexso
```
Comment (louis-jan) [inline README.md:125]: @irfanpena please help separate npm and brew installation into 2 sections. I mean cortex installed by npm would not be uninstalled using brew, so better to not group by action.

Comment (irfanpena) [reply README.md]: This is to make it concise. I have updated and included the uninstall into the quickstart section to separate it. Kindly check @louis-jan

File: cortex-js/src/app.ts [hunk 8]
```diff
@@ -23,7 +23,7 @@ export const getApp = async () => {
   const config = new DocumentBuilder()
     .setTitle('Cortex API')
     .setDescription(
-      'Cortex API provides a command-line interface (CLI) for seamless interaction with large language models (LLMs). Fully compatible with the [OpenAI API](https://platform.openai.com/docs/api-reference), it enables straightforward command execution and management of LLM interactions.',
+      'Cortex API provides a command-line interface (CLI) for seamless interaction with large language models (LLMs). It is fully compatible with the [OpenAI API](https://platform.openai.com/docs/api-reference) and enables straightforward command execution and management of LLM interactions. Cortex = `http://localhost:1337/api`. Jan = `http://localhost:1337/v1`',
```
Comment (louis-jan) [inline cortex-js/src/app.ts:26]: Please help rebase, I think the file is relocated

File: cortex-js/src/app.ts [hunk 9]
```diff
@@ -63,7 +63,6 @@ export const getApp = async () => {
       'Endpoints for  observing Cortex statuses through event notifications.',
     )
     .addServer('http://localhost:1337')
-    .addServer('http://localhost:1337/v1')
```
Comment (louis-jan) [inline cortex-js/src/app.ts:66]: I thought you tried to add 
```ts
.addServer('http://localhost:1337/api')
.addServer('http://localhost:1337/v1')
```

File: cortex-js/src/app.ts (hunk 8 above)
Comment (irfanpena) [reply cortex-js/src/app.ts]: @louis-jan please check, Am I doing the rebase correctly?

File: cortex-js/src/app.ts (hunk 9 above)
Comment (irfanpena) [reply cortex-js/src/app.ts:66]: No, those are incorrect

**Review by louis-jan — APPROVED** [review]
LGTM

**Comment by irfanpena** [comment]
@louis-jan Added the `v1` slug URL. Also I cannot merge the PR on my own

**Pull request merged**